samples/SmokeTest/SmokeTest/SmokeTest.csproj

```diff
@@ -2,16 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;net461</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.3" />
-    <PackageReference Include="Azure.Messaging.EventHubs" Version="5.0.0-preview.2" />
-    <PackageReference Include="Azure.Security.Keyvault.Secrets" Version="4.0.0-preview.3" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.0.0-preview.2" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB" Version="2.4.1" />
-    <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.4.1" />
+    <PackageReference Include="Azure.Identity" Version="1.0.0-preview.4" />
```

Comment:
These versions are all duplicated in https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Packages.Data.props. We should try and have this project get the versions from there. 

I think if we set this property (https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.props#L39) and import this sdk (https://github.com/Azure/azure-sdk-for-net/blob/master/eng/Directory.Build.Data.targets#L2) we will enforce this project has the latest shipped versions we list in that file like any other project in this repo would be forced to do.

Reply:
I manually override versions for `Microsoft.Azure.DocumentDB.Core` and `System.Reflection.Metadata` because I don't think Smoke Tests should set these values for the whole repo. 

I did put `Azure.Security.Keyvault.Secrets` in the `Packages.Data.props` file because that is the released version and it seems appropriate.